pycon/static/less/bootstrap/mixins.less

```diff
@@ -558,13 +558,13 @@
   .core (@gridColumnWidth, @gridGutterWidth) {
```

Comment:
Oh I see comments above. I have no idea what this is about either. Maybe the server needs a newer version of `lessc`?